src/main/resources/config/application.yml

```diff
@@ -179,6 +179,16 @@ application:
         delay: 1000 #in milliseconds
         multiplier: 2
 
+    specification-name: dashboardspec.yml
+    specification-path-pattern: /config/tenants/{tenantName}/dashboard/${application.specification-name}
+    storage:
+        store-configuration-enabled: false
```

Comment:
suggest rename `store-configuration-enabled` to `store-ms-config-enabled` or move enable flag to `ms-config` container like: 
```
ms-config:
   enabled: true
   ...
```

Reply:
if `store-configuration-enabled: true` ms should get the storage type from the dashboardspec.yml spec (RDBMS, MSCONFG). This property doesn't indicate the storage type is MSCONFG